test.py

```diff
@@ -84,19 +86,6 @@ def test_submitSendEmailRequest(self):
         res_json = json.loads(response.data.decode('ascii'))
         self.assertEqual(res_json['code'], 1)
 
-class SubmitUserInfoTests(unittest.TestCase):
```

Comment:
Why remove this test?

Reply:
Because it is tested in the API of `/API/submitUserInfo`